packages/eslint-plugin/src/stylex-valid-styles.js

```diff
@@ -50,6 +50,7 @@ export type RuleCheck = (
   node: $ReadOnly<Expression | Pattern>,
   variables?: Variables,
   prop?: $ReadOnly<Property>,
+  key?: string,
```

Comment:
I made this argument optional because I don't know how `RuleCheck` is being used.

Should I make it required?

Reply:
This OK to start. We can make this stricter in the future if it makes sense.